### PR TITLE
neutron-l3-ha-service: log to a file and reduce verbosity

### PR DESCRIPTION
**Pull request by abel-navarro**
These log lines are output every 10s. Reduced verbosity to 'debug':

- checking for dead agents
- supervisor: starting Subprocess
- supervisor: monitoring Subprocess
- supervisor: done running Subprocess
- error counter: re-set to 0
- Sleeping until next run

Also, log to a separated file in order not to mix with `/var/log/messages`.

Reference: https://bugzilla.suse.com/show_bug.cgi?id=1059460

### CODE REVIEW

**Comment by abel-navarro** [comment]
[Bug 1059460 neutron-l3-ha-service logging fix](https://trello.com/c/GfalPbOW/116-bug-1059460-neutron-l3-ha-service-logging-fix)

File: chef/cookbooks/neutron/files/default/neutron-l3-ha-service.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@
 class HAToolLog
   def self.log
     if @logger.nil?
-      @logger = Logger.new(STDERR)
+      @logger = Logger.new('/var/log/neutron/neutron-l3-ha-service.log', 'daily')
```
[the remark this answers is not in the document]
Comment (nicolasbock) [reply chef/cookbooks/neutron/files/default/neutron-l3-ha-service.rb]: Please double quote this string for hound's sake. Thanks!

**Comment by Itxaka** [comment]
If hound fixed, +1 from me

**Comment by abel-navarro** [comment]
This PR has been superseded by https://github.com/crowbar/crowbar-openstack/pull/1343

**Pull request closed without merging**